test/coverlet.collector.tests/CoverletCoverageDataCollectorTests.cs

```diff
@@ -74,7 +75,10 @@ public void OnSessionStartShouldPrepareModulesForCoverage()
                     null,
                     _context);
             IDictionary<string, object> sessionStartProperties = new Dictionary<string, object>();
-            Coverage coverage = new Coverage("abc.dll", null, null, null, null, null, true, true, "abc.json", true, It.IsAny<ILogger>(), (IInstrumentationHelper)DependencyInjection.Current.GetService(typeof(IInstrumentationHelper)), (IFileSystem)DependencyInjection.Current.GetService(typeof(IFileSystem)));
+            Coverage coverage = new Coverage("abc.dll", null, null, null, null, null, true, true, "abc.json", true,
```

Comment:
please keep declaration in one line...it's not so long at the moment, I prefer all in one line or all stacked.

Reply:
You are right consistency is better, fixed.